build/bazel/remote/execution/v0.2/remote_execution.proto

```diff
@@ -269,6 +269,21 @@ service ContentAddressableStorage {
   }
 }
 
+// The Capabilities service may be used by remote execution clients to query
+// various server properties, in order to self-configure or return meaningful
+// error messages.
+// The query may include a particular `instance_name`, in which case the values
```

Comment:
Nit: paragraph break should be a blank line, or else the lines should be wrapped together.

Reply:
Done.